gstd/src/common/handlers.rs

```diff
@@ -45,29 +45,38 @@ mod panic_handler {
     #[cfg(feature = "debug")]
     #[panic_handler]
     pub fn panic(panic_info: &PanicInfo) -> ! {
-        use crate::prelude::format;
-        #[cfg(not(feature = "panic-messages"))]
-        let message = None::<&core::fmt::Arguments<'_>>;
-        #[cfg(feature = "panic-messages")]
-        let message = panic_info.message();
+        use crate::prelude::{borrow::Cow, format, ToString};
```

Comment:
That's pretty similar approach we used to have before using nightly feature.
Please, keep the feature for that.

Reply:
done

Reply:
I meant introduction of different feature, that is not nightly and can't be used with nightly one

Reply:
but why I should create feature for that instead of just fallback to stable rust?